doc/nrf/app_dev/device_guides/nrf70/features.rst

```diff
@@ -49,6 +49,9 @@ The nRF70 Series devices also support the following functionalities:
 
 * :ref:`ug_nrf70_developing_raw_ieee_80211_packet_transmission`: Allows the injection of raw IEEE 802.11 frames in Station and Monitor modes.
 * :ref:`Promiscuous reception <ug_nrf70_developing_promiscuous_packet_reception>`: Allows the reception of IEEE 802.11 packets from a connected BSSID when operating in Station mode.
+* :ref:`Wi-Fi advanced security modes <ug_nrf70_wifi_advanced_security_modes>`: Allows the use of certificate-based Wi-Fi security and the Platform Security Architecture (PSA) security framework.
+* :ref:`Offloaded raw transmission <ug_nrf70_developing_offloaded_raw_tx>`: Allows the offloading of raw IEEE 802.11 frame transmission to the nRF Wi-Fi driver.
+* :ref:`Wi-i advanced security modes <ug_nrf70_wifi_advanced_security_modes>`: Allows the use of advanced security modes, such as Wi-Fi enterprise security and  PSA crypto support.
```

Comment:
Consider combining line 52 and 54 as follows, since it is about the same document so would not be useful to have the same link twice.
"* :ref:`Wi-Fi advanced security modes <ug_nrf70_wifi_advanced_security_modes>`: Allows the use of advanced security modes, certificate-based Wi-Fi security, and the Platform Security Architecture (PSA) security framework."